sql/moz-fx-data-shared-prod/ads_derived/desktop_tiles_forecast_inputs_v1/metadata.yaml

```
@@ -14,10 +14,10 @@ labels:
 scheduling:
   dag_name: bqetl_ads
   destination_table: >-
-    desktop_tiles_forecast_inputs_v1${{ds.strftime("%Y%m01")}}
+    desktop_tiles_forecast_inputs_v1${{execution_date.strftime("%Y%m01")}}
```

`execution_date` is deprecated but `logical_date` should be equivalent
https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#deprecated-variables
```suggestion
    desktop_tiles_forecast_inputs_v1${{logical_date.strftime("%Y%m01")}}
```